src/components/ModalBuyMealNew/styles.module.scss

```diff
@@ -0,0 +1,202 @@
+@import '../../index.scss';
+@import '../../responsive';
+
+.amountContainer {
+  background-color: #f7f7f7;
+  padding: 25px 35px;
```

Comment:
this is a nit and we have no standard for this — but most people do either multiples of 2, 5 or 8px's. I notice that we don't have any rules. What do you think we should do in our codebase? @bvillaroman @ath-chen